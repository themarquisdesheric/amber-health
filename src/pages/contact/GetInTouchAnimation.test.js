import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import GetInTouchAnimation from './GetInTouchAnimation';

test('GetInTouchAnimation', () => {
  const wrapper = shallow(<GetInTouchAnimation />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
