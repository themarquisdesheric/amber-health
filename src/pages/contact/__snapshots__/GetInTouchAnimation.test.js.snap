// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GetInTouchAnimation 1`] = `
Array [
  <span
    key="G-0"
    style={
      Object {
        "animation": "flipUp 1s cubic-bezier(0.14, -0.4, 0.26, 1.55) 0.1s backwards",
      }
    }
  >
    G
  </span>,
  <span
    key="e-1"
    style={
      Object {
        "animation": "flipUp 1s cubic-bezier(0.14, -0.4, 0.26, 1.55) 0.2s backwards",
      }
    }
  >
    e
  </span>,
  <span
    key="t-2"
    style={
      Object {
        "animation": "flipUp 1s cubic-bezier(0.14, -0.4, 0.26, 1.55) 0.3s backwards",
      }
    }
  >
    t
  </span>,
  <span
    key=" -3"
    style={
      Object {
        "animation": "flipUp 1s cubic-bezier(0.14, -0.4, 0.26, 1.55) 0.4s backwards",
      }
    }
  >
     
  </span>,
  <span
    key="i-4"
    style={
      Object {
        "animation": "flipUp 1s cubic-bezier(0.14, -0.4, 0.26, 1.55) 0.5s backwards",
      }
    }
  >
    i
  </span>,
  <span
    key="n-5"
    style={
      Object {
        "animation": "flipUp 1s cubic-bezier(0.14, -0.4, 0.26, 1.55) 0.6s backwards",
      }
    }
  >
    n
  </span>,
  <span
    key=" -6"
    style={
      Object {
        "animation": "flipUp 1s cubic-bezier(0.14, -0.4, 0.26, 1.55) 0.7s backwards",
      }
    }
  >
     
  </span>,
  <span
    key="t-7"
    style={
      Object {
        "animation": "flipUp 1s cubic-bezier(0.14, -0.4, 0.26, 1.55) 0.8s backwards",
      }
    }
  >
    t
  </span>,
  <span
    key="o-8"
    style={
      Object {
        "animation": "flipUp 1s cubic-bezier(0.14, -0.4, 0.26, 1.55) 0.9s backwards",
      }
    }
  >
    o
  </span>,
  <span
    key="u-9"
    style={
      Object {
        "animation": "flipUp 1s cubic-bezier(0.14, -0.4, 0.26, 1.55) 1s backwards",
      }
    }
  >
    u
  </span>,
  <span
    key="c-10"
    style={
      Object {
        "animation": "flipUp 1s cubic-bezier(0.14, -0.4, 0.26, 1.55) 1.1s backwards",
      }
    }
  >
    c
  </span>,
  <span
    key="h-11"
    style={
      Object {
        "animation": "flipUp 1s cubic-bezier(0.14, -0.4, 0.26, 1.55) 1.2s backwards",
      }
    }
  >
    h
  </span>,
  <span
    key=".-12"
    style={
      Object {
        "animation": "flipUp 1s cubic-bezier(0.14, -0.4, 0.26, 1.55) 1.3s backwards",
      }
    }
  >
    .
  </span>,
]
`;
